prompt_lean_setup

```diff
@@ -18,6 +18,8 @@ PROMPT_LEAN_PATH_PERCENT=${PROMPT_LEAN_PATH_PERCENT-60}
 PROMPT_LEAN_NOTITLE=${PROMPT_LEAN_NOTITLE-0}
 PROMPT_LEAN_CMD_MAX_EXEC_TIME=5
 PROMPT_LEAN_ABBR_METHOD=${PROMPT_LEAN_ABBR_METHOD-"truncate"}
+PROMPT_LEAN_NOVCS=${PROMPT_LEAN_NOVCS-0}
```

Comment:
probably more readable to revert this `PROMPT_LEAN_VCS`=1 is vcs and 0 is not, and we default to 1